.npmignore

```diff
@@ -1,6 +1,5 @@
 # Don't ignore lib
 
-src
```

Comment:
Could we change this to the following?

```
src
!src/schematics/ng-add/schema.json
```

Or perhaps you could switch from npmignore to the package.json files, if safelisting is easier than excluding files.